src/libraries/QueryProcessor.sol

```diff
@@ -66,24 +82,27 @@ library QueryProcessor {
      *  - if the buffer is empty.
      *  - if querying past information and the buffer has not been fully initialized.
      *  - if querying older information than available in the buffer. Note that a full buffer guarantees queries for the
-     *    past 34 hours will not revert.
+     *    past largest safe query window will not revert.
      *
      * If requesting information for a timestamp later than the latest one, it is extrapolated using the latest
      * available data.
      *
      * When no exact information is available for the requested past timestamp (as usually happens, since at most one
      * timestamp is stored every two minutes), it is estimated by performing linear interpolation using the closest
-     * values. This process is guaranteed to complete performing at most 10 storage reads.
+     * values. This process is guaranteed to complete performing at most 11 storage reads.
      */
     function getPastAccumulator(ReservoirPair pair, Variable variable, uint16 latestIndex, uint256 ago)
-        public
+        internal
         view
         returns (int256)
     {
         // solhint-disable not-rely-on-time
         // `ago` must not be before the epoch.
         if (block.timestamp < ago) revert InvalidSeconds();
-        uint256 lookUpTime = block.timestamp - ago;
+        uint256 lookUpTime;
+        unchecked {
+            lookUpTime = block.timestamp - ago;
+        }
```

Comment:
Does this need a safety comment, same for the others missing a safety comment?

Reply:
for this particular one, no. Because two lines prior we checked that `block.timestamp >= ago`. 

So we can be assured that this will not underflow. Do you think that it should still be explicitly documented? 

For the rest, they are accompanying comments but maybe they don't all start with `// safety:`. Should we standardize it everywhere?

Reply:
I would standardize on:
```solidity
// SAFETY:
//
// Comment
```

Reply:
okay added `SAFETY` comments for unchecked blocks